Userland/Libraries/LibC/complex.h

```diff
@@ -42,4 +46,40 @@ long double cimagl(long double complex z);
 #define cimagf(z) ((float)__imag__((float complex)z))
 #define cimagl(z) ((long double)__imag__((long double complex)z))
 
+// Function definitions of this form "type (name)(args)" are intentional, to
+// prevent macro versions of "name" from being incorrectly expanded. These
+// functions are here to provide external linkage to their macro implementations.
+
+// https://pubs.opengroup.org/onlinepubs/9699919799/functions/creal.html
+float(crealf)(float complex z)
```

Comment:
What's going on here? surely we can't put non-inline function declarations in a header file. Why do we need to drop the .cpp file?

Reply:
Would defining them in a .c file work? That's a curious workaround. If it doesn't then I suppose we can keep them `inline`.